src/utils/applyTheme.ts

```diff
@@ -1,20 +1,17 @@
 import { internalThemeResolver } from "../internal/utils/internalThemeResolver.ts";
-import { DevLogger } from "./devLogger.ts";
 import { SavedTheme, ThemeConfig } from "./themeResolver.ts";
 
 export type Document = {
   [key: string]: any;
 };
 
 const THEME_STYLE_TAG_ID = "visual-editor-theme";
-const devLogger = new DevLogger();
 
 export const applyTheme = (
   document: Document,
   themeConfig: ThemeConfig,
   base?: string
 ): string => {
-  devLogger.logFunc("applyTheme");
```

Comment:
the devlogger object would need to be passed into these functions so I just removed it

Reply:
You can't create it at the top like it was before?

Reply:
Updated to use a static variable so that we can create new DevLogger() anywhere vs. needing to prop drill from the place where it receives the query param